tests/stt/pages/databaseChecksPage.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   // insert your locators and methods here
   // setting locators
   url: 'graph/pmm-database-checks',
+  oldUrl: 'graph/d/pmm-checks/pmm-database-checks',
```

Comment:
add comment, what is oldUrl, because you and me we both are getting old and would forget it :stuck_out_tongue:

Reply:
lol. Sure